src/assets/translations/en/main.json

```diff
@@ -1265,7 +1265,7 @@
     },
     "send": {
       "consolidate": {
-        "body": "ShapeShift honors %{asset}'s security design by using a new address for each transaction. To align with THORChain's protocol, which operates with a single address, we'll consolidate your funds into a single address.",
+        "body": "ShapeShift honors %{asset}'s security design by using a new address for each transaction. To align with %{protocolName}'s protocol, which operates with a single address, we'll consolidate your funds into a single address.",
```

Comment:
Not THORChain specific anymore! Relay needs sweep too!

Reply:
Reverted in [8aad17b](https://github.com/shapeshift/web/pull/9222/commits/8aad17b5bc9be5a528d329651d22fa59044b852e)